code/modules/organs/internal/lungs/filter.dm

```diff
@@ -3,6 +3,8 @@
 	name = "advanced lungs"
 	removed_type = /obj/item/organ/lungs/filter
 
+	min_bruised_damage = 30
+	min_broken_damage = 60
```

Comment:
I tested it and in every test case you died way, way before you ever had to deal with your lungs rupturing.  Which is as good as removing it entirely.